src/expression.h

```diff
@@ -112,6 +113,7 @@ class Range : public Expression
 	ValuePtr evaluate(const class Context *context) const;
 	virtual void print(std::ostream &stream) const;
 private:
+    virtual bool isLiteral() const { return true;}
```

Comment:
It also is wrong if we can do this thing.
```
a=1;
b=10;
c=[1:10];
```
But I didn't create I widget. So, I resume their is no need for check like in vector.
Plus behavior of ranges with start and end as variable is wierd.